modules/consentManagement.js

```diff
@@ -337,6 +349,15 @@ export function setConfig(config) {
     utils.logInfo(`consentManagement config did not specify allowAuctionWithoutConsent.  Using system default setting (${DEFAULT_ALLOW_AUCTION_WO_CONSENT}).`);
   }
 
+  if (userCMP === 'static') {
+    if (utils.isPlainObject(config.consentData)) {
+      staticConsentData = config.consentData;
+      consentTimeout = 0;
+    } else {
+      utils.logInfo(`consentManagement config with cmpApi: 'static' did not specify consentData. No consents will be available to adapters.`);
```

Comment:
This should likely be an `logWarn` or `logError` instead of a `logInfo` type message; as this is a negative situation that should be highlighted/easy to spot for a dev troubleshooting.

Reply:
Thanks, will fix!